Tests/ConstantsTests.swift

```diff
@@ -0,0 +1,28 @@
+import XCTest
+@testable import PushNotifications
+
+class ConstantsTests: XCTestCase {
```

Comment:
I think these tests are a bit silly, but I guess it makes sense to prevent people to accidentally change their values without realizing that changing them will impact existing SDK users.

🤷‍♂️